Sources/Components/ChristmasWishListView/Views/MissionPopupView.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+public protocol MissionPopupViewDelegate: AnyObject {
+    func missionPopupViewDidSelectClose(_ view: MissionPopupView)
+}
+
+/// An attempt to create a generic pop-up like container to create views like the klimabrølet pop up
+public class MissionPopupView: UIView {
```

Comment:
Why did you choose this name?

Reply:
Because Thuy knows the component like this